portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,47 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Antonio!</h1>");
+    ArrayList<String> data = createArrayList();
+    String json = convertToJson(data);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Creates an ArrayList with example data.
+   * @return an ArrayList of strings with hardcoded data.
+   */
+  public static ArrayList<String> createArrayList() {
```

Comment:
Can this method be private? 

Quoting from the Java style guide: "Aside from the classes and members you are intentionally designing to be exposed as a public API, all remaining classes and members should use the lowest visibility possible."